localization/i18n/es/OrcaSlicer_es.po

```diff
@@ -2232,16 +2238,16 @@ msgid_plural "Following model objects have been repaired"
 msgstr[0] "Se ha reparado el siguiente modelo de objeto"
 msgstr[1] "Se han reparado los siguientes objetos del modelo"
 
-msgid "Failed to repair following model object"
-msgid_plural "Failed to repair following model objects"
+msgid "Failed to repair folowing model object"
```

Comment:
This should stay as following in both cases.

Reply:
@SoftFever should correct it in original PO.